docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -190,6 +195,8 @@ Json Parameters:
 -   **OpenStdin** - Boolean value, opens stdin,
 -   **StdinOnce** - Boolean value, close stdin after the 1 attached client disconnects.
 -   **Env** - A list of environment variables in the form of `VAR=value`
+-   **Labels** - A map of labels and their values that will be added to the
```

Comment:
Adds a map of labels that to a container. To specify a map: `{"name":"value"[,"name2":"value2"]}`